x-pack/platform/plugins/shared/fleet/public/applications/integrations/sections/epm/screens/installed_integrations/components/confirm_bulk_uninstall_modal.tsx

```diff
@@ -18,23 +19,40 @@ export const ConfirmBulkUninstallModal: React.FunctionComponent<{
 }> = ({ onClose, onConfirm, selectedItems }) => {
   const [isLoading, setIsLoading] = useState(false);
 
+  const isSingleItem = selectedItems.length === 1;
+
   return (
     <EuiConfirmModal
-      title={i18n.translate('xpack.fleet.installedIntegrations.bulkUninstallModal.title', {
-        defaultMessage: 'Uninstall {countIntegrations} integrations ',
-        values: {
-          countIntegrations: selectedItems.length,
-        },
-      })}
+      title={
+        isSingleItem
+          ? i18n.translate('xpack.fleet.installedIntegrations.bulkUninstallModal.singleTitle', {
+              defaultMessage: 'Uninstall {integrationName} ',
+              values: {
+                integrationName: selectedItems[0].title,
+              },
+            })
+          : i18n.translate('xpack.fleet.installedIntegrations.bulkUninstallModal.title', {
+              defaultMessage: 'Uninstall {countIntegrations} integrations ',
+              values: {
+                countIntegrations: selectedItems.length,
+              },
+            })
+      }
       confirmButtonText={i18n.translate(
         'xpack.fleet.installedIntegrations.bulkUninstallModal.confirmButton',
         { defaultMessage: 'Uninstall integrations' }
```

Comment:
should be `Uninstall integration` or `Uninstall {integrationName}` for single selection